app/mailers/submission_mailer.rb

```diff
@@ -9,10 +9,9 @@ def ready_for_cataloging
     @etds_uncataloged_all = Submission.at_ils_loaded.where.not(druid: Settings.skip_cataloging_alert)
                                       .order(:folio_instance_hrid)
 
+    return unless mailable_environment?
```

Comment:
Trivial nit: move this to the first line of the method so we don't even bother with the DB queries if we already know we're gonna short-circuit?